mobilecoind-json/src/bin/main.rs

```diff
@@ -641,10 +641,10 @@ fn main() {
                 balance,
                 utxos,
                 public_address,
-                get_request_code,
-                read_request_code,
-                get_address_code,
-                read_address_code,
```

Comment:
I think there will need to be README updates as well

Reply:
These are function names. The URLs haven't changed.

Reply:
Ah I see - sorry, I think you just went through this same comment loop on another PR today as well!